docs/design/authentication-patterns.md

```diff
@@ -0,0 +1,68 @@
+# Authentication
+
+Use  dialogs to present authentication screens to your end users. Our flows provide a seamless experience that integrates the Microsoft sign-in with your brand.
+
+## Best practices
+
+<br/>
+
+|Do          |Don't       |
+|:-----------|:-----------|
+| Utilize the space to reinforce your company's brand. Azure Active Directory allows customization of sign-in and access panel pages for consistent look and feel with your services. [Learn More](https://azure.microsoft.com/en-us/documentation/articles/active-directory-add-company-branding/).| Don't overcrowd the task pane or dialogs with buttons.|
+Target your dialogs to a few key widths or breakpoints for better responsive design. [Learn more](https://msdn.microsoft.com/windows/uwp/layout/screen-sizes-and-breakpoints-for-responsive-design)|                      
```

Comment:
The guideline about target widths/breakpoints and responsive design seem not that relevant to this topic. Seems to me to be more relevant in a topic on dialogs.